src/database/models/roommodel.js

```diff
@@ -0,0 +1,21 @@
+
+module.exports = (sequelize, DataTypes) => {
+  const RoomModel = sequelize.define('RoomModel',{
+   roomType: DataTypes.STRING,
+   description: DataTypes.STRING,
+   roomLabel: DataTypes.STRING,
+   hotelId: DataTypes.INTEGER,
+   status: DataTypes.STRING,
+   price: DataTypes.STRING,
+   roomImage: DataTypes.STRING
+  },
+  {
+   sequelize,
+   modelName: 'RoomModel',
+   onUpdate: 'CASCADE',
+   onDelete: 'CASCADE'
+ });
+
+  
+   return RoomModel;
+ };
```

Comment:
Please consider defining associations of since rooms will belong to hotels

Reply:
Associations now defined